app/views/content/funding-and-support.md

```diff
@@ -2,9 +2,9 @@
 title: "Teacher training funding"
 heading: "Fund your teacher training"
 description: |-
-    Explore funding for teacher training, including student finance, bursaries and scholarships, and support if you’re disabled, a parent or carer or a veteran.
+    Find out about teacher training costs, student finance, bursaries and scholarships, and funding if you’re disabled, a parent or carer or a veteran.
 title_paragraph: |-
-    If you're applying for teacher training in England, we can help you understand what funding and practical support you could be eligible for.
+    If you're applying for teacher training in England, we can help you understand the difference between courses with fees and courses with salaries, and what funding and practical support you could be eligible for.
```

Comment:
```suggestion
    If you're applying for teacher training in England, we can help you understand the cost of teacher training and what funding and practical support you could be eligible for.
```

I'm not sure we are helping people to understand the overall differences between courses with fees and salaries? It's also quite a long para. However, I'm not sure my alternative works as it mentions cost!